scripts/speech_recognition/convert_to_tarred_audio_dataset.py

```diff
@@ -322,6 +327,15 @@ def create_new_dataset(self, manifest_path: str, target_dir: str = "./tarred/",
                 for i, (start_idx, end_idx) in enumerate(zip(start_indices, end_indices))
             )
 
+        if config.shard_manifests:
+            for manifest in new_entries_list:
+                shard_id = manifest[0]['shard_id']
+                new_manifest_shard_path = os.path.join(target_dir, f'manifest_{shard_id}.json')
+                with open(new_manifest_shard_path, 'w') as m2:
```

Comment:
Ii wonder if it's not better to save both 1 main manifest and the create a subdir with all of the shared manifest files by default. 
It build a shard manifest in subdir always  and use inverse flag to avoid creating it.

Reply:
changed to --no_shard_manifests which turns off the creation of sharded manifests (which is now true by default).